articles/libraries/auth0js/migration-guide.md

```diff
@@ -0,0 +1,112 @@
+---
+section: libraries
+description: How to migrate from auth0.js v7 to auth0.js v8
+url: /libraries/auth0js/migration-guide
+---
+
+# Auth0.js v7 to v8 Migration Guide
+
+The following instructions assume you are migrating from **auth0.js v7** to **auth0.js v8**. 
+
+The goal of this migration guide is to provide you with all of the information you would need to update Auth0.js in your application. Of course, your first step is to include the latest version of auth0.js. Beyond that, take a careful look at each of the areas on this page. You will need to change your implementation of auth0.js to reflect the new changes. Note that we especially recommend migrating to auth0.js v8 if you have the OAuth2 flag enabled in your tenant. 
+
+Take a look below for more information about changes and additions to auth0.js in version 8!
+
+## Initialization of auth0.js
+
+Initialization of auth0.js in your application will now use `auth0.WebAuth` instead of `Auth0`
+```html
+<script src="https://cdn.auth0.com/js/auth0/8.0.4/auth0.min.js"></script>
+<script type="text/javascript">
+  var webAuth = new auth0.WebAuth({
+    domain:       'YOUR_AUTH0_DOMAIN',
+    clientID:     'YOUR_CLIENT_ID'
+  });
+</script>
+```
+
+## Login
+
+The `login` method of version 7 was divided into several different methods in version 8, based on the type of auth you need, rather than the old `login` method.
+
+### webAuth.authorize()
+
+The `authorize` method can be used for logging in users via the [Hosted Login Page](/libraries/auth0js#hosted-login-page), or via social connections, as exhibited below. The default behavior for `authorize` is redirect, but popup can be specified.
```

Comment:
`.authorize()` just redirects and you can't tell it to use a popup. There is another method inside the `popup` "namespace" to perform popup auth like you explained later. What I mean is they are different methods instead of a single one